### PR TITLE
create_host_config: allow setting oom_kill_disable

### PR DESCRIPTION
**Pull request by schu**

### CODE REVIEW

File: docker/utils/utils.py [hunk 1]
```diff
@@ -511,6 +511,9 @@ def create_host_config(
     if privileged:
         host_config['Privileged'] = privileged
 
+    if oom_kill_disable:
```
Comment (shin-) [inline docker/utils/utils.py]: Can we test that `version_gte(version, '1.19') is True and raise an `InvalidVersion` otherwise?

**Comment by shin-** [comment]
Thanks, looks good! Just a small request to make it perfect :+1:

File: docker/utils/utils.py (hunk 1 above)
Comment (schu) [inline docker/utils/utils.py]: Done.

**Comment by shin-** [comment]
Thanks!

LGTM.

**Comment by schu** [comment]
Thanks!

**Pull request merged**